src/components/Modal/Modal.styled.jsx

```diff
@@ -1,6 +1,7 @@
 import styled from '@emotion/styled';
 
 export const Overlay = styled.div`
+  z-index: 99999999;
```

Comment:
This is for..
![localhost_3000_delivery_902b7f07-da34-4ab5-9a8c-d9db03ba55b(BlackBerry Z30) (1)](https://user-images.githubusercontent.com/71513679/159194054-562922c0-e8a4-4da9-b25f-1772f22e27f5.png)
![localhost_3000_delivery_902b7f07-da34-4ab5-9a8c-d9db03ba55b(BlackBerry Z30)](https://user-images.githubusercontent.com/71513679/159194057-bc9d2a16-048f-4fc1-ab74-2853f326aacb.png)
.

Reply:
A little too much, but effective hehe